scripts/clean_appliance.py

```diff
@@ -44,8 +44,8 @@ def main():
         # `systemctl stop evmserverd` is a little slow, and we're destroying the
         # db, so rudely killing ruby speeds things up significantly
         print('Stopping ruby processes...')
-        ssh_client.run_command('killall ruby')
-        ssh_client.run_rake_command('evm:db:reset')
+        ssh_client.run_command('systemctl stop evmserverd')
```

Comment:
@quarckster From what I tried on my appliance (5.9.0.17) it actually takes quite a lof of time for `killall ruby` to kill MIQ Server process. Try to `killall ruby` and then check few times `systemctl status evmserverd`. You will find out that MIQ Server process is active for a long time. And when you try to execute `DISABLE_DATABASE_ENVIRONMENT_CHECK=1 bin/rake -f /var/www/miq/vmdb/Rakefile evm:db:reset` while MIQ Server is up, you will be not allowed to do that. 

Then try the same with `systemctl stop evmserverd`. All processes will be stop in much quicker fashion. I don't know why, but this is quicker and cleaner way to do it. 

Of course I should edit/delete surrounding comments accordingly.

Reply:
@mshriver I can see that it's your change, could you please take a look here?

Reply:
heh, my only commit is actually just changing this to a context managed ssh_client. 

https://github.com/ManageIQ/integration_tests/pull/4657/commits/77375c13c29459cfe7286537caf4f26529b7dad5

If `killall ruby` is taking a long time, and we don't care if the process shutdown is non-graceful, can we pass a SIGKILL here instead of the default SIGTERM to speed it up?

Reply:
The other option is to add a dynamic 'wait' before the `evm:db:reset`, looking for evmserverd status to change before proceeding.

Reply:
I think I have to go with `systemctl stop evmserverd`. There is one problem with `killall` command. There is a process `evm_watchdog.rb` and even if you send SIGKILL to it, the process gets indeed killed but respawns right after that. I have not managed to find a way around that, so I think I'll have to stop the processes nicely. 

If you execute `systemctl stop evmserverd` on the server, `systemctl` command waits for the stop. Therefore I am not sure if `wait_for` call is needed. What do you think about it, @quarckster, @mshriver ?

Reply:
Honestly I like graceful shutdown more.

Reply:
@jan-zmeskal Thanks for digging into this thoroughly! I'd agree with @quarckster that graceful shutdown is probably preferable even if we're going to trash the DB.

Given your investigation here, I would like to see an update to the in-line comments at this block of code. Otherwise they're then out-of-date and in direct conflict with the actual code. Please describe (in-line comments) why a clean shutdown is done, and why ruby processes are restarted otherwise.

Please ping me when you've done so and it should be a quick review/merge then :)